modules/meeting/app/controllers/meeting_contents_controller.rb

```diff
@@ -92,8 +92,8 @@ def diff
 
   def notify
     unless @content.new_record?
-      service = MeetingNotificationService.new(@meeting, @content_type)
-      result = service.call(@content, :content_for_review)
+      service = MeetingNotificationService.new(@meeting)
+      result = service.call(:invited)
```

Comment:
The purpose of this notification seems to be changed from agenda review email to invited notification. We still have the button for sending content review email, but it will do a different action. Should we still keep that review functionality around, or rather delete the review button?

The button is rendered from the [meeting_contents_helper.rb](https://github.com/opf/openproject/blob/92df5058e7716e327e0f2123e187f49c45f5a341/modules/meeting/app/helpers/meeting_contents_helper.rb#L49): 

```ruby
    # ...
    if saved_meeting_content_text_present?(content)
      menu << meeting_content_notify_link(content_type, content.meeting)
      menu << meeting_content_icalendar_link(content_type, content.meeting)
    end
    # ...
```

<img width="1056" alt="image" src="https://github.com/opf/openproject/assets/83396/59acc784-a653-4647-8365-c8b87736f5eb">

Reply:
Very good point. I have moved them to a "More" menu in the toolbar, similar to how the structured meetings do it. We don't need it at the agenda or meeting level, as they are not containing any content specific data.